frameworks/helloworld/tests/test_sanity.py

```diff
@@ -19,21 +21,65 @@
 
 
 @pytest.fixture(scope="module", autouse=True)
-def configure_package(configure_security):
+def helloworld_service(configure_security):
```

Comment:
we are using `configure_package` (just grepped and it has 39 occurences) and these methods usually assume `install` of the package. any strong reason for changing this to `helloworld_service` now?

Reply:
No real reason. I thought it's more communicative with respect to the configuration that is being passed through. I will revert this change for the time being.

Reply:
3fcf4de